ts/Series/Heatmap/HeatmapSeries.ts

```diff
@@ -120,6 +120,10 @@ class HeatmapSeries extends ScatterSeries {
 
         /**
          * The border radius for each heatmap item.
+         * The border's color and width can be set in marker options.
```

Comment:
One more thing: please do not break lines in one paragraph, see [style guidelines](https://github.com/highcharts/highcharts/blob/master/repo-guidelines.md#style-guide)